dist/s_clang_format

```diff
@@ -43,17 +20,12 @@ cd "$dest_dir" || exit 1
 export PATH="${PWD}/dist":$PATH
 
 # Check if Clang-Format is already available with the desired version.
-desired_version="10.0.0"
-if ! command -v clang-format &> /dev/null; then
-    download_clang_format $desired_version || exit 1
-elif ! clang-format --version | grep -q $desired_version; then
-    download_clang_format $desired_version || exit 1
+desired_version="12.0.1"
+if ! command -v clang-format &> /dev/null || ! clang-format --version | grep -q $desired_version; then
+    echo "Failed to find Clang-Format, make sure the binary (version: $desired_version) is in your env PATH."
```

Comment:
Clang-Format --> clang-format (to better match the binary name)? 
As there are 2 conditions covered by the same error message, is it worth mentioning install the package (besides putting the binary under env PATH)?

Reply:
Let me know if 1037fe0fabd228c0acf913d81c977eba7bde8052 works for you